### PR TITLE
Adds WFJT Details view

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This addresses #5814.

It adds the WFJT detail view along with delete and launch functionality. It also uses introduces a new look utilizing Labels for resources that should be linked out.  See #5977 

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION
```
9.2.1
```

##### ADDITIONAL INFORMATION
Screen Shot 2020-02-18 at 12 01 51 PM

### CODE REVIEW

File: awx/ui_next/src/components/DetailList/DetailList.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const DetailList = ({ children, stacked, ...props }) => (
 export default styled(DetailList)`
   display: grid;
   grid-gap: 20px;
-  align-items: flex-start;
+  align-items: center;
```
Comment (AlexSCorey) [inline awx/ui_next/src/components/DetailList/DetailList.jsx:14]: On a detailView when a cell requires 2 lines that label becomes off centered.  This was introduce to center the label, vertically.

File: awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import CardCloseButton from '@components/CardCloseButton';
 import ContentError from '@components/ContentError';
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx:9]: This is ready to be refactored into a functional component, which I can/plan to do. However, I wanted to keep the scope of this PR smaller.

File: awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx [hunk 3]
```diff
@@ -110,6 +132,10 @@ class WorkflowJobTemplate extends Component {
                 <WorkflowJobTemplateDetail
                   hasTemplateLoading={hasContentLoading}
                   template={template}
+                  webHookKey={webHookKey}
+                  onSetContentLoading={() =>
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx]: I plan to remove this in favor of allowing the Detail View to handle it's own content loading state.

File: awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx [hunk 4]
```diff
@@ -1,17 +1,244 @@
...
+import DeleteButton from '@components/DeleteButton';
+import { toTitleCase } from '@util/strings';
+import { Sparkline } from '@components/Sparkline';
+
+function WorkflowJobTemplateDetail({
+  template,
+  hasContentLoading,
+  i18n,
+  webHookKey,
+  onSetContentLoading,
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx]: Remove this.  Allow this component to manage it own loading state

File: awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx (hunk 2 above)
Comment (jlmitch5) [reply awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx:9]: 👍 JT Form also needs functionalized.  I started doing it when I was moving thigns to the new useField formik hook, but it started getting a lot bigger, so for similar reasons, I decided to hold off on that.

File: awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx [hunk 5]
```diff
@@ -1,17 +1,239 @@
...
+        <span> {i18n._(t`(Prompt on Launch)`)}</span>
+      </>
+    ) : (
+      <Link to={`/inventories/${inventorykind}/${inventoryId}/details`}>
+        <Label>{summary_fields.inventory.name}</Label>
+      </Link>
+    );
+  };
+  const canLaunch =
+    summary_fields.user_capabilities && summary_fields.user_capabilities.start;
```
Comment (jlmitch5) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx]: do you wanna do that cool ?. stuff you used above here?

**Review by jlmitch5 — APPROVED** [review]
looks great!

File: awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx (hunk 5 above)
Comment (AlexSCorey) [reply awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx]: oh, yeah I should do that here.

**Pull request merged**